cmd/miner/cpuminer.go

```diff
@@ -89,7 +88,10 @@ func (m *CPUMiner) hashRateMonitor(ctx context.Context) {
 // This function will return early with false when conditions that trigger a
 // stale block such as a new block showing up or periodically when there are
 // new transactions and enough time has elapsed without finding a solution.
-func (m *CPUMiner) solveBlock(ctx context.Context, headerB []byte, target *big.Rat, ticker *time.Ticker) bool {
+func (m *CPUMiner) solveBlock(ctx context.Context, headerB []byte, target *big.Rat) bool {
+	ticker := time.NewTicker(333 * time.Millisecond)
```

Comment:
This ticker seems to be unused (I think it was also unused before this PR)

Reply:
The ticker is used here https://github.com/decred/dcrpool/blob/399537d35c0d81933a4dab53e68d182a388c0a5b/cmd/miner/cpuminer.go#L116, it was passed in as an argument before.

Reply:
My mistake, I overlooked that.